examples/cms-strapi/next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  images: {
+    domains: ['localhost'],
```

Comment:
This is dangerous.

Did you mean to use `process.env.NEXT_PUBLIC_STRAPI_API_URL` instead?